l10n_it_fatturapa_in/models/attachment.py

```diff
@@ -113,12 +121,11 @@ def _compute_xml_data(self):
                             in SELF_INVOICE_TYPES:
                         att.is_self_invoice = True
                 att.invoices_date = ' '.join(invoices_date)
-            else:
-                att.xml_supplier_id = False
-                att.invoices_number = 0
-                att.invoices_total = 0
-                att.is_self_invoice = False
-                att.invoices_date = False
+            except Exception as e:
+                _logger.error(
```

Comment:
Forse l'errore nel log non gli va?
```suggestion
                _logger.warning(
```